documentation/staging/content/userguide/managing-operators/troubleshooting.md

```diff
@@ -90,19 +97,65 @@ $ kubectl -n OP_NAMESPACE describe pod weblogic-operator-UNIQUESUFFIX
 ```
 A pod `describe` usefully includes any events that might be associated with the operator.
 
-### Check common issues
+### Check the conversion webhook deployment
+
+Verify that the conversion webhook's deployment is deployed and running by listing all deployments with the `weblogic.webhookName` label.
+
+```text
+$ kubectl get deployment --all-namespaces=true -l weblogic.webhookName
+```
+
+Check the conversion webhook deployment's detailed status:
+
+```text
+$ kubectl -n WH_NAMESPACE get deployments/weblogic-operator-webhook -o yaml
+```
+
+And/or:
+
+```text
+$ kubectl -n WH_NAMESPACE describe deployments/weblogic-operator-webhook
+```
+
+Each conversion webhook deployment will have a corresponding Kubernetes pod
+with a name that has a prefix that matches the deployment name,
+plus a unique suffix that changes every time the deployment restarts.
+
+To find conversion webhook pods and check their high-level status:
+
+```text
+$ kubectl get pods --all-namespaces=true -l weblogic.webhookName
+```
+
+To check the details for a given pod:
+
+```text
+$ kubectl -n WH_NAMESPACE get pod weblogic-operator-webhook-UNIQUESUFFIX -o yaml
+$ kubectl -n WH_NAMESPACE describe pod weblogic-operator-webhook-UNIQUESUFFIX
+```
+A pod `describe` usefully includes any events that might be associated with the conversion webhook.
```

Comment:
Add something like: For information about installing and uninstalling the webhook, see ...